<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketWidget</class>
 <widget class="QWidget" name="TicketWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продажа/возврат билетов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="3">
    <widget class="QLineEdit" name="lineEdit_flight"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Номер паспорта</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Номер билета</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Номер авиарейса</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_ticket"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_passport"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_sell">
     <property name="text">
      <string>Продать билет</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_return">
     <property name="text">
      <string>Вернуть билет</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QCheckBox" name="checkBox_search">
     <property name="text">
      <string>Отфильтровать данные</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4">
    <widget class="QTableView" name="tableView_tickets"/>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QTableView" name="tableView_passengers"/>
   </item>
   <item row="3" column="0" rowspan="5">
    <widget class="QTableView" name="tableView_flights"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
